packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -902,11 +903,18 @@ export default class InternalModel {
 
   notifyHasManyChange(key: string) {
```

Comment:
Thank you so much @runspired. I would never have guessed the culprit was here. I don't know how you found that. All look so easy for you 😅